### ISSUE TITLE
Earth Spirit - Stone Remnant bug after 7.32c patch

### ISSUE THREAD

**Issue by SavanDR7** [ValveSoftware/Dota2-Gameplay#3492]
### Ability name

stone remnant

### Description

when you click on stone remnant double application, the stone is placed very far from the character, so it does not immediately roll quickly and because of this it is impossible to hit it properly

### Example Match ID (and possibly Timestamp)

6779988080 all time

**Comment by homephix1** [comment]
Any solution for the problem[.](https://homephix.co.uk/) I have recokned that the problem is due to a generic bug. Please let me know if anyone find the solution. Thank you.

**Comment by Acuvter** [comment]
yeah im also having the same problem and its especially annoying when trying to stun enemies in front of you

**Comment by Noz23** [comment]
Agreed, really annoying bug when you use double tap for roll... hope this get fixed

**Comment by andersaas** [comment]
Dumb sloppy fix by valve.

**Comment by yakongxing** [comment]
valve，please fix this！
